recipes/cruft/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "cruft" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 300a2f19dfd94e44a66f5b68d179a2a57b03c2fc7bbc25367d5b5f5c26681e0f
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-cache-dir -vvv"
```

Comment:
It looks like this package contains an entry point. Please add it:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

Also add a `commands` test in the `test` section below:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands